### DISCUSSION TITLE
Can the Mtex provide a typical random-misorientation-distribution?

### DISCUSSION THREAD

**Discussion by Brian-Liu-rx** [mtex-toolbox/mtex#1758] [Ask Anything]
Hi, everyone,

Can the Mtex provide a typical random-misorientation-distribution data?

**Comment by ralfHielscher** [comment]
yes :)

```
orientation.rand(100000,cs1,cs2)
```

**Reply by Brian-Liu-rx** [reply]
Thank you rallf. But how can I get their boundary misorientation distribution?

**Comment by lfgmorales** [comment]
Hi

You mean something like the red line in the last plot?

Cheers

Luiz

Luiz F. G. Morales
ScopeM - ETH Zürich
Sent from iPhone

On 10 Mar 2023, at 03:16, Brian-Liu-rx ***@***.***> wrote:

﻿

Thank you rallf. But how can I get their boundary misorientation distribution?

—
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/1758#discussioncomment-5261979>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5IN3CMJMJ2A24I6IA7FDW3KFHXANCNFSM6AAAAAAVVGIJOU>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/1758#discussioncomment-5261979", "url": "https://github.com/mtex-toolbox/mtex/discussions/1758#discussioncomment-5261979", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]

**Reply by Brian-Liu-rx** [reply]
![image](https://user-images.githubusercontent.com/69452222/224242470-a4eff6b3-f621-4109-a937-7896b37b7136.png)

**Reply by Brian-Liu-rx** [reply]
Hi Luiz,
just like the black line

**Reply by Brian-Liu-rx** [reply]
I make it. I use the getdataw.exe tool to capture the data of the black curve：））））

![image](https://user-images.githubusercontent.com/69452222/224247519-d45465bf-dc4e-4992-b07e-f9a92f27ab7d.png)
